inception-app-webapp/pom.xml

```diff
@@ -127,6 +127,11 @@
       <groupId>de.tudarmstadt.ukp.inception.app</groupId>
       <artifactId>inception-external-search-elastic</artifactId>
     </dependency>
+      <dependency>
+          <groupId>de.tudarmstadt.ukp.inception.app</groupId>
+          <artifactId>inception-external-search-solr</artifactId>
+          <version>0.16.0-SNAPSHOT</version>
```

Comment:
Versions should me managed in the top-level POM. Individual modules should not declare versions for their dependencies.

Indentation.